tests/integration/init/schemas/schemas_test_data_setup.py

```diff
@@ -55,17 +55,43 @@ def setUpClass(cls):
             runner = CliRunner()
             runner.invoke(init_cmd, ["--output-dir", temp], input=user_input)
 
-    def _init_custom_config(self, profile, region):
-        self.config_dir = tempfile.mkdtemp()
         env = os.environ
         if AWS_CONFIG_FILE in env:
-            self.original_config_file = env[AWS_CONFIG_FILE]
+            cls.original_config_file = env[AWS_CONFIG_FILE]
         if AWS_SHARED_CREDENTIALS_FILE in env:
-            self.original_cred_file = env[AWS_SHARED_CREDENTIALS_FILE]
+            cls.original_cred_file = env[AWS_SHARED_CREDENTIALS_FILE]
         if AWS_PROFILE in env:
-            self.original_profile = env[AWS_PROFILE]
+            cls.original_profile = env[AWS_PROFILE]
         if AWS_DEFAULT_REGION in env:
-            self.original_region = env[AWS_DEFAULT_REGION]
+            cls.original_region = env[AWS_DEFAULT_REGION]
+
+    @classmethod
+    def tearDownClass(cls) -> None:
```

Comment:
Can we remove `_tear_down_custom_config` or call it from here given that they have pretty similar logic?

Reply:
_tear_down_custom_config helps keep each test case's configuration clean, so I don't really want to touch that, one refactoring that I can do tho, is to apply this tear down as a test case tearDown method into every test case instead of a class method, some extra resets, but we can remove _tear_down_custom_config that way